Modules/selectmodule.c

```diff
@@ -335,6 +335,10 @@ select_select_impl(PyObject *module, PyObject *rlist, PyObject *wlist,
         if (tvp) {
             timeout = deadline - _PyTime_GetMonotonicClock();
             if (timeout < 0) {
+                /* lists were unmodified -- we have to clear them explicitly */
```

Comment:
One last thing, would you mind to add "bpo-35310: " prefix to this comment? It would help for readers in the future.